config.yaml

```
@@ -63,7 +63,9 @@ params:
     - title: Documentation
       url: https://docs.pybamm.org/
       is_external: true
-
+    - title: GitHub
+      url: https://github.com/pybamm-team/
```

```suggestion
      url: https://github.com/pybamm-team/PyBaMM
```
In #171, it was asked to have link to the repo.